system-tests/tests/buildiso-@

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Check that Cobbler is able to build customized ISOs by the "cobbler buildiso" command with the addition of the
+# airgapped flag
+
+source ${SYSTESTS_PRELUDE} && prepare
+
+build_iso_test=${TEST_NAME#buildiso-}
+
+trap cleanup EXIT
+
+cleanup() {
+	mountpoint -q ${mp} && umount ${mp}
+	rmdir ${mp}
+}
+
+wget -P "${tmp}/" https://download.opensuse.org/distribution/leap/15.3/iso/openSUSE-Leap-15.3-DVD-x86_64-Current.iso
+mp=$(mktemp -dt leap-mp-XXX)
+mount -o loop,ro "${tmp}/openSUSE-Leap-15.3-DVD-x86_64-Current.iso" "${mp}"
+cobbler import --name leap --path "${mp}"
+
+set -x -e -o pipefail
+
+# Preparations
+cobbler system add --name testbed --profile leap-x86_64
+
+# Tmp: Create "/var/cache/cobbler" because it does not exist per default
+mkdir -p /var/cache/cobbler/buildiso
+
+# Real test
+case ${build_iso_test} in
+    airgapped)
+        cobbler buildiso --airgapped \
+                         --distro=leap-x86_64 \
+                         --tempdir="/var/cache/cobbler/buildiso" \
+                         --iso ${tmp}/autoinst.iso
+        ;;
+    full)
+        cobbler buildiso --standalone \
+                         --distro=leap-x86_64 \
+                         --tempdir="/var/cache/cobbler/buildiso" \
+                         --iso ${tmp}/autoinst.iso
+        ;;
+    net)
+        cobbler buildiso --tempdir="/var/cache/cobbler/buildiso" --iso ${tmp}/autoinst.iso
+        ;;
+esac
+
+# Check ISO exists
+[ -f ${tmp}/autoinst.iso ]
```

Comment:
Redundant now since xorriso will fail in case autoinst.iso is missing.

Reply:
True. Fixed via force push.